pgmpy/factors/distributions/ContinuousDistribution.py

```diff
@@ -0,0 +1,230 @@
+from pgmpy.factors.distributions import BaseDistribution
+import numpy as np
+from scipy import integrate
```

Comment:
Please refer to PEP8 (https://www.python.org/dev/peps/pep-0008/#imports) for import order and also group them.